script.js

```diff
@@ -0,0 +1,144 @@
+const myLibrary = [];
+
+const body = document.querySelector('body');
+
+// Book constructor
+function Book(title, author, pages, status) {
+  this.title = title;
+  this.author = author;
+  this.pages = pages;
+  this.status = status;
+}
+function toggle() {
+  this.status = !this.status;
+}
+Book.prototype.toggleRead = toggle;
+
+function displayBooks() {
+  if (document.querySelector('table')) {
+    body.removeChild(document.querySelector('table'));
+  }
+
+  const table = document.createElement('table');
+  const rowHead = document.createElement('tr');
+  const colHeadTitle = document.createElement('th');
+  colHeadTitle.textContent = 'Title';
+  rowHead.appendChild(colHeadTitle);
+  const colHeadAuthor = document.createElement('th');
+  colHeadAuthor.textContent = 'Author';
+  rowHead.appendChild(colHeadAuthor);
+  const colHeadPages = document.createElement('th');
+  colHeadPages.textContent = 'Pages';
+  rowHead.appendChild(colHeadPages);
+  const colHeadStatus = document.createElement('th');
+  colHeadStatus.textContent = 'Read Status';
+  rowHead.appendChild(colHeadStatus);
+
+  body.appendChild(table);
+
+  for (let i = 0; i < myLibrary.length; i += 1) {
+    const row = document.createElement('tr');
+
+    const title = document.createElement('td');
+    title.textContent = myLibrary[i].title;
+    row.appendChild(title);
+
+    const author = document.createElement('td');
+    author.textContent = myLibrary[i].author;
```

Comment:
## Modulirise this module
- [ ] Let's keep application logic separated from DOM manipulation tasks ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/dom_logic_separation_requirement.png) from Pathwright)